lib/routes.js

```diff
@@ -19,7 +21,28 @@ module.exports = function (server) {
     method: 'POST',
     path: '/signup',
     handler: function (request, reply) {
-      reply.file('index.html');
+      var email = request.payload.email + '';
+      var optin = request.payload.optin + '';
+
+      var emailCheck = !!email.match(/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/i);
```

Comment:
Let's import `Joi` as to keep this consistent with how we do email validation in the API / Login:
https://github.com/hapijs/joi#stringemailoptions